main.go

```diff
@@ -30,6 +30,7 @@ func main() {
 	}
 
 	err = systempreferences.ConfigureDock(cfg.MacOS.Dock)
+	err = systempreferences.ConfigureDateAndTime(cfg.MacOS.DateAndTime)
```

Comment:
We need to check for the error above. I am seeing pattern here that there will be more `systempreferences` calls so we can make a little helper function to check errors, and since we are using `log.Fatalf()` we don't need to return anything (because _gobee_ will exit if error occurs).

```go
func ok(err) {
	if err != nil {
		log.Fatalf("Failed to apply configuration. %s\n", err)
	}
}
```

---

It may be a bit overkill now since we have only two calls to `systempreferences`. So simple copy-paste code that checks `err` will be enough. I prefer copy-pasta now.

Reply:
I see the execution needing to be more dynamic than is currently allowing. I'm going to merge and move on to address this somehow.